book/text.md

```diff
@@ -293,22 +312,24 @@ line spacing is a normal amount.
 
 ::: {.further}
 Breaking lines in the middle of a word is called hyphenation, and can
-be turned on via the [`hyphens` CSS property][hyphens]. Browsers
-use the [Knuth-Liang hyphenation algorithm][liang], which uses a
-dictionary of word fragments to prioritize possible hyphenation
-points, to implement this.
+be turned on via the [`hyphens` CSS property][hyphens]. The state of
+the art is the [Knuth-Liang hyphenation algorithm][liang], which uses
+a dictionary of word fragments to prioritize possible hyphenation
+points, to implement this. However, it's not clear that the CSS
+specification [allows browsers to use this algorithm][css-hyphen].
 :::
 
 [liang]: http://www.tug.org/docs/liang/liang-thesis.pdf
 [hyphens]: https://drafts.csswg.org/css-text-3/#hyphens-property
+[css-hyphen]: https://news.ycombinator.com/item?id=19472922
```

Comment:
Obsoleted by https://groups.google.com/a/chromium.org/g/blink-dev/c/rwBWqqOB_ag, can you update the text accordingly?